src/__tests__/core/binanceSignTransaction.spec.js

```diff
@@ -0,0 +1,154 @@
+/* @flow */
+
+import type {
+    TestFunction,
+    SubtestSignTransaction,
+} from 'flowtype/tests';
+
+import type {
+    TestCardanoSignTransactionPayload,
```

Comment:
This looks weird - importing CardanoTypes into Binance code?

Reply:
solved, thanks